src/controllers/ArticleLikeController.js

```diff
@@ -0,0 +1,31 @@
+import statusCode from '../config/status';
+import db from '../models';
+import updateArticleLikes from '../helpers/updateArticleLikes';
+
+/**
+ * A class to control the Article likes
+ */
+export default class ArticleLikeController {
+  /**
+   * A method to create an article like
+   * @param {object} req the request to get like
+   * @param {object} res the response of like from server
+   * @returns {object} the return after liking a particular article
+   */
+  static async createLike(req, res) {
+    let message = 'You likes the article';
+    const userId = req.user.id;
+    const { articleSlug, status } = req.params;
+    const createLike = await db.ArticleLike.create({
+      userId,
+      articleSlug,
+      status
+    });
+
+    updateArticleLikes(req);
+    if (status === 'dislike') {
+      message = 'You disliked the article';
```

Comment:
and here

Reply:
I think a message must be short, and easy to understand. As the user will know the article they are liking there is no need to display the slug in the message..